fedmsg_meta_fedora_infrastructure/tests/pagure.py

```diff
@@ -1623,6 +1623,70 @@ class TestGitCommit(Base):
          'topic': 'io.pagure.prod.pagure.git.receive'
     }
 
+class TestIssueDrop(Base):
+    """ These messages are published when a ticket is closed against a
```

Comment:
I think it means `delete` .
https://github.com/pypingou/pagure/blob/e1497f5e98387fa547e251ccf02ba179adc95202/pagure/ui/issues.py#L532